test/net/sourceforge/kolmafia/preferences/AlsoPreferencesTest.java

```diff
@@ -0,0 +1,85 @@
+package net.sourceforge.kolmafia.preferences;
+
+import static internal.helpers.Player.withProperty;
+import static internal.helpers.Player.withSavePreferencesToFile;
+import static internal.helpers.Utilities.verboseDelete;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import internal.helpers.Cleanups;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import net.java.dev.spellcast.utilities.DataUtilities;
+import net.sourceforge.kolmafia.KoLCharacter;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AlsoPreferencesTest {
```

Comment:
why has this been split out?

Reply:
Because?  :-)  In trying to get test cleanups that actually deleted files I realized that these were the only tests that needed the preference file to persist long enough that the backup (and possibly restore) code kicked in.  What was cleaned up, and when, seemed to differ when I un-nested them.  And, while I don't think it hurt anything the before and after code is different here compared to the rest of the tests.  It also removed a possible source of conflict because it used a different username and explicitly made it lower case compared to the other tests.

I had my reasons but since they were part of an evolutionary process, I will try putting these back in the other file and see what breaks.  I'll mark this as Draft until I have done so or can tell you what is not working.  Thank you.

Reply:
Merged